lagunita/lms/templates/static_templates/tos.html

```diff
@@ -2,7 +2,6 @@
 
 <%!
   from django.conf import settings
-  from django.core.urlresolvers import reverse
```

Comment:
I think a number of the variables in these static templates are also copy/pasta relics, like concern_email, privacy_email, etc..
Can you sweep these files for other outdated references?

Reply:
this is done